js/OTPALayer.js

```diff
@@ -91,16 +91,29 @@ L.OTPALayer = L.FeatureGroup.extend({
       onEachFeature: onEachPoint(self._filteredPointsetStyle)
     }).addTo(map);
 
+    this._surfaceLayer = null;
+
     this._isochronesLayer = L.geoJson([], {
+      /*
+      style: {
+        color: 'red',
+        smoothFactor: 0.5,
+        noClip: true
+      }
+      */
       style: function(feature) {
         var style = {
+          smoothFactor: 0.7,
+          noClip: true,
           color: '#333',
           fillColor: '#333',
           lineCap: 'round',
           lineJoin: 'round',
           weight: 2,
-          dashArray: '5, 4',
-          fillOpacity: '0.08'
+          //dashArray: '5, 4',
+          fillOpacity: '0.1',
+          smoothFactor: 0.25,
+          noClip: true,
```

Comment:
Yup, I sure did. Neither is really helpful; I wanted to make sure that what I was seeing wasn't Leaflet messing with things by simplifying geometries (that wasn't it). I'll take them both out.